src/blocks/HeaderBlock/MenuItem.tsx

```diff
@@ -273,14 +274,22 @@ export const MenuItem: React.FunctionComponent<{ menuItem: MenuItemType }> = ({
                     isClosing={isClosing}
                     hasMenuGroups={true}
                   >
-                    {menu_item_groups.map(({ label, menu_items, _uid }) => (
-                      <DropdownMenuItemList key={_uid}>
-                        <GroupLabelListItem>
-                          <MenuGroupLabel>{label}</MenuGroupLabel>
-                        </GroupLabelListItem>
-                        <SubMenuLinks menu_items={menu_items} />
-                      </DropdownMenuItemList>
-                    ))}
+                    {menu_item_groups.map(
+                      ({ label, link, menu_items, _uid }) => (
+                        <DropdownMenuItemList key={_uid}>
+                          <GroupLabelListItem>
+                            {link?.cached_url ? (
+                              <MenuGroupLabel href={getStoryblokLinkUrl(link)}>
+                                {label}
+                              </MenuGroupLabel>
+                            ) : (
+                              <MenuGroupLabel as="span">{label}</MenuGroupLabel>
```

Comment:
Is this being rendered as a `span`? I mean, if I understood correctly, `MenuGroupLabel` accepts a prop called `as` which can be `undefined | 'span'` but it does nothing with it, right?

Reply:
Ah yes! The `as` prop changes the rendered tag, it's a prop for styled components. https://emotion.sh/docs/styled#as-prop But I needed to add the type for it.
The alternative is creating a new style component like so:
```
const MenuGroupLabel = styled(MenuGroupLink)({ cursor: 'default' }).withComponent(
  'span',
)
```